src/utils/perpareReciterSurahsList.js

```diff
@@ -13,7 +13,7 @@ export default (reciter, chapters, chapterId) => {
       id: chapters[index].id,
       path: `https://download.quranicaudio.com/quran/${reciter.relative_path}${zeroFill(3, chapters[index].id)}.mp3`,
       title: chapters[index].name.simple,
-      artist: reciter.name,
+      artist: reciter.name || 'hello',
```

Comment:
btw we need to move all that static strings in the app to a Class that holds them as constants. We might also use this package in the future to support localization https://github.com/stefalda/ReactNativeLocalization